buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -16,7 +16,7 @@ object Versions {
 
   const val org_jetbrains_kotlinx_kotlinx_coroutines: String = "1.4.1"
 
-  const val androidx_test_espresso: String = "3.4.0"
```

Comment:
hi @gouri-panda , this is required for android 13 test failure , please have a look at https://github.com/kiwix/kiwix-android/commit/f69aa3521a5dc6f214ef51513a4f55cd5bb1ace8

Reply:
@MohitMaliFtechiz Thanks for the review. @kelson42 What should we do? Should we wait for an alternative solution?

Reply:
Nonclue, but if this PR is a dead end, we shoukd close it.